packages/sdk/typescript/human-protocol-sdk/src/base.ts

```diff
@@ -21,4 +23,19 @@ export abstract class BaseEthersClient {
     this.networkData = networkData;
     this.runner = runner;
   }
+
+  /**
+   * Internal helper to enrich transaction overrides with network specific defaults
+   * (e.g. fixed gas price for Aurora networks) while preserving user provided
+   * fee parameters.
+   */
+  protected applyTxDefaults(txOptions: Overrides = {}): Overrides {
+    if (
+      this.networkData.chainId === ChainId.AURORA_TESTNET &&
+      txOptions.gasPrice === undefined
```

Comment:
A couple questions:
1. Are we planning to include only aurora testnet support?
2. I'm not sure that we need to check if the gasPrice is undefined here. If I remember correctly, every transaction on aurora will cost 0.07 gwei so I don't think we need to give other users to set the gasPrice. In case it's lower that 0.07 transaction might fail, if it's higher than it's an unnecessary spending. @portuu3 please correct me if I'm wrong.

Reply:
1. Aurora mainnet is not in the SDK yet, we will add it later when we deploye the contracts in mainnet
2. You are right. Actually if the user wants to spend is up to him, but also makes sense to just force the static value